CMakeLists.txt

```diff
@@ -195,6 +195,7 @@ include(external/gtest)     # download, build, install gtest
 include(external/protobuf)  # download, build, install protobuf
 include(external/python)    # download, build, install python
 include(external/openblas)  # download, build, install openblas
+include(external/opencv)    # download, build, install opencv
```

Comment:
- please do not add `external/opencv`, could you provide the preprocess data such as `vis_demo.cc` do?
- You can `run.sh`, and the download package which has both model and data.
- And we don't need `image_reader.cc`